src/routes/components/calloutLogic.js

```diff
@@ -0,0 +1,12 @@
+
+import { get } from 'lodash';
+import showToast from './showToast';
+
+export default function calloutLogic(props, prevProps, keyString, calloutFunc) {
```

Comment:
Possibly don't need this utility function?

Reply:
I'm conflicted, because I get that we might not need this function, and it's only in one place but equally it's lovely and concise, and makes the `componentDidUpdate` method much shorter too. The thing that's making me think that this should perhaps stay for now is that there are plans to have deletion/creation callouts on all modules, and I think this logic is going to be very similar for those, so it might eventually be worth adding to `stripes-erm-components` or something... Idk, I'd leave it up to you as the reviewer, but I'm thinking right now it does more good than harm.

Reply:
This function seems to handle for example calls only from componentDidUpdate as it would need `prevProps`. We may need to call the callout from a different function where we wont have access to `prevProps` and that means we need to write additional logic again to handle those case.

I would say after you conditionally call the `calloutLogic` from within the `CDU` you could actually have the `calloutLogic` as another function within the `JobsRoute` itself for now, that way `CDU` would be cleaner. and we could eventually write another shared component/util function.

Reply:
This makes sense, I'll move it over.

I'm not sure what you mean about adding code to deal with the case where there's no `prevProps`, This is very specifically just for when a change is made to a job, ie on `creation/deletion/edit` (edit is potentially something we could add in the future). For all of those don't we require the `prevProps` offered by `CDU` to check whether things happened?